package.json

```diff
@@ -32,7 +32,7 @@
     "localforage-observable": "1.4.0",
     "mark.js": "8.11.1",
     "marked": "github:mattermost/marked#e57e07f419f85178354a3bbbd60de5302483c436",
-    "mattermost-redux": "github:mattermost/mattermost-redux#6bddbe0f9331b011e71e9e0aaff47702e4a0ece4",
```

Comment:
This reference is pointing to the `mattermost-redux` branch commit just so the tests pass, it will be changed to the redux merge commit in `master` before merging.